cartography/data/jobs/analysis/aws_lambda_function_asset_exposure.json

```diff
@@ -10,14 +10,49 @@
 			"iterative": true,
 			"iterationsize": 100
 		},
+		{
```

Comment:
public facing does not apply to function alias or layer. please remove the code related to alias and layer.